tests/test_cert_handling.sh

```diff
@@ -68,6 +71,18 @@ if [[ $(docker run --rm  --entrypoint stat -v clientvar:/var nivlheimclient -c "
 	exit 1
 fi
 
+# Verify that the certificate contains Subject Alternative Name, and that it matches Common Name
+docker run --rm  --entrypoint openssl -v clientvar:/var nivlheimclient x509 -in /var/nivlheim/my.crt -noout -text > $tempdir/cert
+NAME=$(sed -n 's/^.*Subject:.*CN\s*=\s*\(\S*\)/\1/p' < $tempdir/cert)
+if [[ "$NAME" == "" ]]; then
+	echo "Didn't find the Common Name in the client certificate."
+	exit 1
+fi
+if ! grep -Pzo "X509v3 extensions:\n\s*X509v3 Subject Alternative Name:\s*\n\s*DNS:$NAME\s*\n" $tempdir/cert; then
```

Comment:
Av nysgjerrighet: hvorfor `-z` til grep her?  Iflg manualen er det eksperimentelt kombinert med `-P`, og vi får morsomme "bokser" i [test output](https://github.com/unioslo/nivlheim/runs/5647363876?check_suite_focus=true).  LGTM uansett. :shipit:

Reply:
`-z` er for at den skal se på hele filen som en linje, for at regexen skal kunne matche på noe som går over flere linjer. Har ikke funnet noen bedre måte å gjøre det.

Reply:
Skjønner, takk for oppklaring.  LGTM :+1: